src/main/java/de/retest/recheck/ui/descriptors/AttributeUtil.java

```diff
@@ -1,9 +1,44 @@
 package de.retest.recheck.ui.descriptors;
 
 import java.awt.Rectangle;
+import java.util.Objects;
+
+import de.retest.recheck.ui.diff.AttributeDifference;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.IdentifyingAttributesDifference;
 
 public class AttributeUtil {
 
+	public static Rectangle getActualOutline( final ElementDifference difference ) {
+		final IdentifyingAttributesDifference identifyingAttributesDifference =
+				(IdentifyingAttributesDifference) difference.getIdentifyingAttributesDifference();
+		try {
+			if ( Objects.nonNull( identifyingAttributesDifference.getAttributeDifferences() ) ) {
```

Comment:
Just use `!= null` since there is no advantage to use `nonNull`. Also extract `identifyingAttributesDifference.getAttributeDifferences()` since you call it twice.